aiohttp/web_response.py

```diff
@@ -155,16 +154,15 @@ def set_status(
         self,
         status: int,
         reason: Optional[str] = None,
-        _RESPONSES: Mapping[int, Tuple[str, str]] = RESPONSES,
```

Comment:
I wonder if this existed to for testing to allow for an easy DI.

Reply:
@Dreamsorcerer thoughts?

Reply:
FWIW, removing this would certainly be a backwards-incompatible change. So I wouldn't include it for this PR to remain backportable.

Reply:
It first appeared in aiohttp v2.0.0: https://github.com/aio-libs/aiohttp/pull/1626/files#diff-1a42e29607eed54e0f8cc46ddec171806240d00d867b9c1dfde1757c57ace744R559. That patch adds this argument to a few other places but I'm not sure if it's ever used.

Reply:
It's undocumented and prefixed with `_`, indicating a private variable.
https://docs.aiohttp.org/en/stable/web_reference.html#aiohttp.web.StreamResponse.set_status

I'd still be inclined to drop it.